jxls-poi/src/test/java/org/jxls3/EachTest.java

```diff
@@ -0,0 +1,218 @@
+        List<Employee> employees = Employee.generateSampleEmployeeData().subList(0, 4);
+        employees.get(0).setName("i");
+        employees.get(1).setName("Z");
+        employees.get(2).setName("A");
+        employees.get(3).setName("Z");
+		data.put("employees", employees);
+
+        // Test
+        Jxls3Tester tester = Jxls3Tester.xlsx(getClass(), "orderBy");
+        tester.test(data, JxlsPoiTemplateFillerBuilder.newInstance());
+        
+        // Verify
+        try (TestWorkbook w = tester.getWorkbook()) {
+            w.selectSheet("Employees");
+            assertEquals("A", w.getCellValueAsString(2, 1)); 
+            assertEquals("Z", w.getCellValueAsString(3, 1)); 
+            assertEquals("Z", w.getCellValueAsString(4, 1)); 
+            assertEquals(1700d, w.getCellValueAsDouble(4, 3), 0.005d); 
+            assertEquals("i", w.getCellValueAsString(5, 1)); 
+        }
+    }
+
+	@Test
+    public void orderByIgnoreCase() throws IOException {
+        // Prepare
+        Map<String,Object> data = new HashMap<>();
+        List<Employee> employees = Employee.generateSampleEmployeeData().subList(0, 3);
+        employees.get(0).setName("i");
+        employees.get(1).setName("Z");
+        employees.get(2).setName("A");
+		data.put("employees", employees);
+
+        // Test
+        Jxls3Tester tester = Jxls3Tester.xlsx(getClass(), "orderByIgnoreCase");
+        tester.test(data, JxlsPoiTemplateFillerBuilder.newInstance());
+        
+        // Verify
+        try (TestWorkbook w = tester.getWorkbook()) {
+            w.selectSheet("Employees");
+            assertEquals("A", w.getCellValueAsString(2, 1)); 
+            assertEquals("i", w.getCellValueAsString(3, 1)); 
+            assertEquals("Z", w.getCellValueAsString(4, 1)); 
+        }
+    }
+
+	// This is also a direction=RIGHT test.
+    @Test
+    public void ignoreProps() throws ParseException {
+    	// Prepare
+		List<EmployeeWithDepartments> employees = new ArrayList<>();
+		employees.add(new EmployeeWithDepartments("Kylie", "1970-Jul-10", 2100, "Main"));
+		employees.add(new EmployeeWithDepartments("Mariah", "1943-Jan-01", 0));
+		employees.add(new EmployeeWithDepartments("Sascha", "1965-Mar-19", 600, "D", "E", "F", "G", "H"));
+		Map<String, Object> data = new HashMap<>();
+        data.put("employees", employees);
+        
+        // Test
+        Jxls3Tester tester = Jxls3Tester.xlsx(getClass(), "ignoreProps");
+        tester.test(data, JxlsPoiTemplateFillerBuilder.newInstance()
+        		.withIgnoreColumnProps(true).withIgnoreRowProps(true)); // Jxls doesn't change row heights and column widths.
+        
+        // Verify
+        try (TestWorkbook w = tester.getWorkbook()) {
+            w.selectSheet("Employees");
+            assertEquals("Sascha", w.getCellValueAsString(4, 1));
+            assertEquals("direction=RIGHT does not work", "H", w.getCellValueAsString(4, 8));
+            assertFalse(w.getColumnWidth(4) == w.getColumnWidth(5));
```

Comment:
Above you use `assertEquals` and here `assertFalse` for unequality. Would not it be better to use `assertNotEquals(w.getColumnWidth(4), w.getColumnWidth(5));` for consistency instead of `assertFalse`?